daal4py/sklearn/monkeypatch/tests/test_monkeypatch.py

```diff
@@ -0,0 +1,66 @@
+#===============================================================================
+# Copyright 2020-2021 Intel Corporation

```

Comment:
```suggestion
# Copyright 2020 Intel Corporation

```

Reply:
I removed old test as it was covered with scope of new test. Please, tell me if you have any objections.
Copyright was also updated.
Thanks =)